twitch/resources.py

```diff
@@ -4,6 +4,9 @@
 
 
 def convert_to_twitch_object(name, data):
+    if data is None:
```

Comment:
From your description of the problem, I understand that the problems are only all `_DateTime` objects and not all object. Instead of doing this here, I'd rather see that we change the `construct_from` method of the `_DateTime` class to return None if the `value is None`.
So the `construct_form` method would then looks something like this:
```
@classmethod
def construct_from(cls, value):
    if value is None:
        return None
    try:
        dt = datetime.strptime(value, '%Y-%m-%dT%H:%M:%SZ')
    except ValueError:
        dt = datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%fZ')
    return dt
```

If we do a generic check for `data is None` like you did here, it will likely hide, fix or introduce some other bugs that are not related to the one we're addressing in this PR.

Reply:
That makes sense, I'll make the suggested change. Thank you!